src/app/applications/english_auction/page.mdx

```diff
@@ -0,0 +1,449 @@
+export const metadata = {
+  title: "English Auction • Stylus by Example",
+  description:
+    "An example implementation of the English Auction in Rust using Arbitrum Stylus.",
+};
+
+{/* Begin Content */}
+
+# English Auction
+
+An Arbitrum Stylus version implementation of [solidity English Auction](https://solidity-by-example.org/app/english-auction/).
+
+## Auction
+- 1. Seller of NFT deploys this contract.
+- 2. Auction lasts for 7 days.
+- 3. Participants can bid by depositing ETH greater than the current highest bidder.
+- 4. All bidders can withdraw their bid if it is not the current highest bid.
+
+## After the auction
+- 1. Highest bidder becomes the new owner of NFT.
+- 2. The seller receives the highest bid of ETH.
+
+Here is the interface for English Auction.
+
+```solidity
+interface IEnglishAuction {
+    function nft() external view returns (address);
+
+    function nftId() external view returns (uint256);
+
+    function seller() external view returns (address);
+
+    function endAt() external view returns (uint256);
+
+    function started() external view returns (bool);
+
+    function ended() external view returns (bool);
+
+    function highestBidder() external view returns (address);
+
+    function highestBid() external view returns (uint256);
+
+    function bids(address bidder) external view returns (uint256);
+
+    function initialize(address nft, uint256 nft_id, uint256 starting_bid) external;
+
+    function start() external;
+
+    function bid() external payable;
+
+    function withdraw() external;
+
+    function end() external;
+
+    error AlreadyInitialized();
+
+    error AlreadyStarted();
+
+    error NotSeller();
+
+    error AuctionEnded();
+
+    error BidTooLow();
+
+    error NotStarted();
+
+    error NotEnded();
+}
+```
+
+### ABI Export
+
+You can export the Solidity ABI for your program by using the `cargo stylus` tool as follows:
+
+```bash
+cargo stylus export-abi
+```
+
+which outputs:
+
+```js
```

Comment:
Since the interface is the same as the one above, I'd say we can remove it from here, and just say that it "outputs the same interface as above".